<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>600</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>400</width>
                <height>600</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>400</width>
                <height>600</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QLabel" name="nameLabel">
                                        <property name="text">
                                            <string>账号 :</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="nameEdit"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_2">
                                <item>
                                    <widget class="QLabel" name="pawdLabel">
                                        <property name="text">
                                            <string>密码 :</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="pawdEdit">
                                        <property name="inputMask">
                                            <string/>
                                        </property>
                                        <property name="text">
                                            <string/>
                                        </property>
                                        <property name="placeholderText">
                                            <string/>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,1">
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                                        <item>
                                            <widget class="QLabel" name="addrLabel">
                                                <property name="text">
                                                    <string>地址 :</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLineEdit" name="addrEdit"/>
                                        </item>
                                    </layout>
                                </item>
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                                        <item>
                                            <widget class="QLabel" name="portLabel">
                                                <property name="maximumSize">
                                                    <size>
                                                        <width>30</width>
                                                        <height>16777215</height>
                                                    </size>
                                                </property>
                                                <property name="text">
                                                    <string>端口 :</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLineEdit" name="portEdit">
                                                <property name="maximumSize">
                                                    <size>
                                                        <width>60</width>
                                                        <height>16777215</height>
                                                    </size>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_6">
                                <item>
                                    <widget class="QLabel" name="label">
                                        <property name="text">
                                            <string extracomment="FTP服务器上的地址">映射 :</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="lineEdit"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="QListWidget" name="listWidget"/>
                        </item>
                        <item>
                            <widget class="QPushButton" name="pushButton">
                                <property name="text">
                                    <string>同步</string>
                                </property>
                                <property name="checkable">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
